account-kit/infra/src/middleware/feeEstimator.ts

```diff
@@ -36,14 +38,23 @@ export const alchemyFeeEstimator: (
   async (struct, { overrides, feeOptions, client: client_ }) => {
     const client = clientHeaderTrack(client_, "alchemyFeeEstimator");
     const transport_ = transport({ chain: client.chain });
-    let [block, maxPriorityFeePerGasEstimate] = await Promise.all([
-      client.getBlock({ blockTag: "latest" }),
-      // it is a fair assumption that if someone is using this Alchemy Middleware, then they are using Alchemy RPC
-      transport_.request({
-        method: "rundler_maxPriorityFeePerGas",
-        params: [],
-      }),
-    ]);
+    const [sender, nonce] = await Promise.all([struct.sender, struct.nonce]);
```

Comment:
❗ ❓ shouldn't we use `overrides.nonce` here if it's set?

Reply:
that gets applied upstream in the middleware stack